blitz.py

```diff
@@ -184,7 +184,10 @@ def cursor_1():
 def jassu_pass_timing() -> int:
     shot_distance = distance(0, 11)
     shot_mod = int(shot_distance / 160)
-    if 540 <= memory.main.get_story_progress() < 570:
+    if (
```

Comment:
I think these two are doing the same thing. Any reason in particular for making the change? Might be worth grabbing `memory.main.get_story_progress()` as a variable so we don't have to call it twice if we end up keeping it the new way.

Reply:
Tbh I didn't think this would work, but it appears it does. Python is weird. I'll revert those particular changes.